tests/foreman/cli/test_import.py

```diff
@@ -218,6 +220,16 @@ def get_sat6_id(
     return entity_dict
 
 
+def wait_for_publish(delay_step=30):
```

Comment:
Non blocker comment: This may be usesful for other cli tests too, may be we can move this to helpers.py